### DISCUSSION TITLE
Can i import an existing adf string to a Document

### DISCUSSION THREAD

**Discussion by jbiscarri** [DamienHarper/adf-tools#3] [Q&A]
Hello, what I need is to convert an adf document string to json/html and get it back to adf string. I'm not sure if with this library is possible or if you know another one that can do the job.
Thanks in advance!

**Comment by T3chW1zard** [comment]
@DamienHarper & @jbiscarri did you find any information you are willing to share regarding the feature?

**Comment by DamienHarper** [comment]
Sorry guys, no HTML importer right now. It would be a quite complex feature as HTML is richer than ADF so many "unsupported" tags/attributes would simply be ignored and the generated ADF document would not be faithful.

**Reply by T3chW1zard** [reply]
It's not about converting HTML to ADF; rather, it's about converting ADF to HTML.

**Reply by DamienHarper** [reply]
Thanks @T3chW1zard to bring light to me, I misread the question, my bad

**Comment by DamienHarper** [comment]
@jbiscarri oops, I misread the question. ADF to json/html is the purpose of this library. 
The [documentation](https://github.com/DamienHarper/adf-tools/blob/main/doc/index.md) might help you (mainly the export sections)
